libfreerdp/core/certificate.c

```diff
@@ -671,7 +671,7 @@ BOOL certificate_read_server_certificate(rdpCertificate* certificate, BYTE* serv
 	if (length < 4) /* NULL certificate is not an error see #1795 */
 		return TRUE;
 
-	s = Stream_New(server_cert, length);
+	s = Stream_New((BYTE*)server_cert, length);
```

Comment:
Not your code, but I guess we could use `Stream_Static_Init` and avoid the the `Stream_Free`in `certificate_read_server_proprietary_certificate` and `certificate_read_server_x509_certificate_chain`.
That could be part of another PR.